net.sourceforge.projects.vht.yml

```diff
@@ -0,0 +1,19 @@
+id: net.sourceforge.projects.vht
```

Comment:
I have pretty sure this is incorrect. The project isn't even hosted on SourceForge. You you did 

> The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

Reply:
This is correct. I have other project there, like zaz

Reply:
This is not. You certainly don't have `projects.sourceforge.net`

Reply:
would github.com.rdybka.vht work?

Reply:
ok, changed it

Reply:
> would github.com.rdybka.vht work?

No

The current io.github one is ok.